src/Method/ResultObject.php

```diff
@@ -111,6 +111,10 @@ public function unserialize($string)
         }
     }
 
+
+    public function __serialize(): array {return[];}
+    public function __unserialize(array $data): void {}
```

Comment:
Is there any reason why we do not reuse `serialize()` and `unserialize()` methods already implemented in this class?

Reply:
It still use the serialize() and unserialize() functions, these __serialize functions are executed prior to the serialize() to do any type of cleanup. 
https://www.php.net/manual/en/class.serializable.php 
Even with no cleanup the definition is needed to avoid deprecated warning.